### PR TITLE
Test1

### PR DESCRIPTION
**Pull request by n1ceGood**

### CODE REVIEW

File: src/com/company/Main.java [hunk 1]
```diff
@@ -1,8 +1,38 @@
...
 
     public static void main(String[] args) {
-        System.out.println("A");
-    }
-}
+
+        Scanner in = new Scanner(System.in);
+        System.out.println("");
+        System.out.println("Enter a");
+        int a = in.nextInt();
```
Comment (costia097) [inline src/com/company/Main.java]: Use Integer instead of int.

File: src/com/company/Main.java [hunk 2]
```diff
@@ -1,8 +1,38 @@
...
+        System.out.println("Enter b");
+        int b = in.nextInt();
```
Comment (costia097) [inline src/com/company/Main.java]: Same here

File: src/com/company/Main.java [hunk 3]
```diff
@@ -1,8 +1,38 @@
...
+        System.out.println("What do you want to do"+"\n"+" Enter '1' = '+';"+"\n"+" Enter '2' = '-';"+"\n"+" Enter '3' = '*';"+"\n"+" Enter '4' = '/';");
+        int d = in.nextInt();
```
Comment (costia097) [inline src/com/company/Main.java]: Here

File: src/com/company/Main.java [hunk 4]
```diff
@@ -1,8 +1,38 @@
...
+        int c ;
```
Comment (costia097) [inline src/com/company/Main.java]: And here

File: src/com/company/Main.java [hunk 5]
```diff
@@ -1,8 +1,38 @@
...
+        if ( d==1){
```
Comment (costia097) [inline src/com/company/Main.java]: Use equals instead == . Better use swich instead of many if.

File: src/com/company/Main.java [hunk 6]
```diff
@@ -1,8 +1,38 @@
...
+            c = a+b;
+            System.out.println(a+"+"+b+"="+c);
+        } else if (d==2){
```
Comment (costia097) [inline src/com/company/Main.java]: Same here

File: src/com/company/Main.java [hunk 7]
```diff
@@ -1,8 +1,38 @@
...
+            c=a-b;
+            System.out.println(a+"-"+b+"="+c);
+        } else if (d==3){
```
Comment (costia097) [inline src/com/company/Main.java]: Here

File: src/com/company/Main.java [hunk 8]
```diff
@@ -1,8 +1,38 @@
...
+            c=a*b;
+            System.out.println(a+"*"+b+"="+c);
+        }else if (d==4){
```
Comment (costia097) [inline src/com/company/Main.java]: Here

**Review by costia097 — CHANGES_REQUESTED** [review]

File: src/com/company/Main.java [hunk 9]
```diff
@@ -1,8 +1,38 @@
...
+            c=a/b;
+            System.out.println(a+"/"+b+"="+c);
+        }
+        else {
+            System.out.println("You did not select an action.");
+        }
+
+            }
+            }
```
Comment (costia097) [inline src/com/company/Main.java]: Reformat it

File: src/com/company/Main.java [hunk 10]
```diff
@@ -1,8 +1,38 @@
...
+            System.out.println(a+"*"+b+"="+c);
+        }else if (d==4){
+            c=a/b;
+            System.out.println(a+"/"+b+"="+c);
+        }
+        else {
+            System.out.println("You did not select an action.");
+        }
+
+            }
```
Comment (costia097) [inline src/com/company/Main.java]: reformat

**Review by costia097 — CHANGES_REQUESTED** [review]

File: src/com/company/Main.java (hunk 10 above)
Comment (n1ceGood) [reply src/com/company/Main.java]: ?

Comment (n1ceGood) [reply src/com/company/Main.java]: What exactly?

**Review by costia097 — APPROVED** [review]

**Pull request merged**